src/config/remote-config.c

```diff
@@ -316,3 +332,21 @@ STATIC int w_remoted_get_net_protocol(const char * content) {
 
     return retval;
 }
+
+int w_read_tcp_config(XML_NODE node, remoted *logr) {
```

Comment:
This function always returns `OS_SUCESS`. Maybe you can return `OS_INVALID` in https://github.com/wazuh/wazuh/pull/18085/commits/f9b5216c508d8c5a44cb203f12a67be946277e57#diff-0e3330f864b11c344a887980e0dcc43cc5ca6a4dbc57ebf7eb3747b30309828dR344 and move the `minfo` after this function is called.

Reply:
The return type of the function was changed to ```void```.